src/ledger/LedgerCloseMetaFrame.cpp

```diff
@@ -219,7 +219,7 @@ LedgerCloseMetaFrame::populateEvictedEntries(
             continue;
         case LEDGER_ENTRY_REMOVED:
             auto const& key = change.removed();
-            releaseAssert(isTemporaryEntry(key));
+            releaseAssert(isTemporaryEntry(key) || key.type() == EXPIRATION);
```

Comment:
the compiler is upset that there's a case missing in the switch enclosing this (LEDGER_ENTRY_CREATED) -- can you add it?